src/Mvc/Extensions.ApiDescription.Design/src/build/Microsoft.Extensions.ApiDescription.Design.targets

```diff
@@ -187,7 +187,7 @@
             Condition="'%(GenerateDefaultDocumentOptions)' == ''">$(GenerateDefaultDocumentDefaultOptions)</GenerateDefaultDocumentOptions>
       </CurrentServiceProjectReference>
       <CurrentServiceProjectReference Update="@(CurrentServiceProjectReference)">
-        <Command>%(Command) --framework %(TargetFramework) --output %(DocumentPath)</Command>
+        <Command>%(Command) --documentName %(DocumentName) --framework %(TargetFramework) --output %(DocumentPath)</Command>
```

Comment:
Assuming these are command line parameters, do these need to be quoted?

Reply:
Probably wouldn't hurt 😺 for `%(DocumentName)`, `%(DocumentPath)` and `%(ProjectExtensionsPath)` (below)